cmd/minikube/cmd/start_flags.go

```diff
@@ -110,6 +110,12 @@ const (
 	network                 = "network"
 	startNamespace          = "namespace"
 	trace                   = "trace"
+	sshIPAddress            = "ip-address"
```

Comment:
how do u feel about `ssh-ip-address`  or `ssh-remote-ip`  ? so all the SSH driver flags start with SSH ? 
I think, it would get lost in the tons of flags we have, if a user is looking only for ssh driver flags...

Reply:
Sure, it got confusing when removed the driver name (generic)